Makefile.pre.in

```diff
@@ -1900,7 +1905,9 @@ BYTESTR_DEPS = \
 		$(srcdir)/Objects/stringlib/partition.h \
 		$(srcdir)/Objects/stringlib/split.h \
 		$(srcdir)/Objects/stringlib/stringdefs.h \
-		$(srcdir)/Objects/stringlib/transmogrify.h
+		$(srcdir)/Objects/stringlib/transmogrify.h \
+		$(srcdir)/Include/internal/pycore_emscripten_trampoline.h \
+		$(srcdir)/Objects/stringlib/find_max_char.h
```

Comment:
Please place it after the existing f prefix (Although the main branch seems to already exist `$(srcdir)/Objects/stringlib/find_max_char.h`, but I am just giving an example).
for example
```
$(srcdir)/Objects/stringlib/fastsearch.h \
$(srcdir)/Objects/stringlib/find.h \
$(srcdir)/Objects/stringlib/find_max_char.h \       
...
```
(Current main branch content, Maybe it's out of date).

**This advice applies to all changes**.